packages/app/components/stackbox/FrequencyOptionsCard.tsx

```diff
@@ -2,13 +2,32 @@
 
 import { ChangeEvent, useEffect, useState } from "react";
 
-import { add } from "date-fns";
+import { add, daysToWeeks } from "date-fns";
 import { twMerge } from "tailwind-merge";
 
+import { useStrategyContext } from "@/contexts";
 import { FREQUENCY_OPTIONS } from "@/models";
 import { BodyText, RadioButton, TextInput } from "@/ui";
 import { cx } from "class-variance-authority";
 
+const parseDaysToFrequencyAmount = (
+  days: number,
+  frequency: FREQUENCY_OPTIONS
+) => {
+  switch (frequency) {
+    case FREQUENCY_OPTIONS.hour:
+      return days * 24;
+    case FREQUENCY_OPTIONS.day:
+      return days;
+    case FREQUENCY_OPTIONS.week:
+      return daysToWeeks(days);
+    case FREQUENCY_OPTIONS.month:
+      return days / 30;
+    default:
+      throw new Error("Invalid frequency option");
```

Comment:
We are throwing an error which is never caught, maybe we should log the error and return something.

Reply:
You're right! I'll console the error and return either the max or min value allowed for the selected frequency.